tests/perf/test_classification.py

```diff
@@ -258,3 +261,121 @@ def test_perf(
             criteria=self.BENCHMARK_CRITERIA,
             resume_from=fxt_resume_from,
         )
+
+
+class TestPerfSemiSLMultiClass(PerfTestBase):
+    """Benchmark single-label classification for Semi-SL task."""
+
+    MODEL_TEST_CASES = [  # noqa: RUF012
+        Benchmark.Model(task="classification/multi_class_cls", name="efficientnet_b0_semisl", category="speed"),
+        Benchmark.Model(task="classification/multi_class_cls", name="mobilenet_v3_large_semisl", category="speed"),
+        Benchmark.Model(task="classification/multi_class_cls", name="efficientnet_v2_semisl", category="accuracy"),
+        Benchmark.Model(task="classification/multi_class_cls", name="deit_tiny_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="dino_v2_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="tv_efficientnet_b3_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="tv_efficientnet_v2_l_semisl", category="other"),
+        Benchmark.Model(task="classification/multi_class_cls", name="tv_mobilenet_v3_small_semisl", category="other"),
+    ]
+
+    DATASET_TEST_CASES = (
+        [
+            Benchmark.Dataset(
+                name=f"cifar10@{num_label}_{idx}",
+                path=Path(f"multiclass_classification/semi-sl/cifar10@{num_label}_{idx}/supervised"),
+                group="cifar10",
```

Comment:
Group should be one of "small", "medium", "large". Could you update it?

Reply:
As far as I know, this is just a str type to collect the results in Summary, so I think it's okay to leave it as it is, but what do you think? Will this cause errors elsewhere?

The dataset used here doesn't have a clear distinction between small, medium, and large.

Reply:
Currently, only "small", "medium", "large" and "all" can be passed to `data_group` arguments. Please check "tests/conftest.py". So, only if "all" is passed to the argument, these datasets won't be skipped. If you think it's ok, then I have objection to leave it as it is.